pyro/optim/optim.py

```diff
@@ -176,15 +176,17 @@ def save(self, filename: str) -> None:
         with open(filename, "wb") as output_file:
             torch.save(self.get_state(), output_file)
 
-    def load(self, filename: str) -> None:
+    def load(self, filename: str, map_location: Optional[str] = None) -> None:
```

Comment:
It looks like `map_location` can be [more general](https://pytorch.org/docs/stable/_modules/torch/serialization.html#load) than a string, so I'd recommend omitting a type hint here. (But thanks for adding a type hint, we do try to add type hints to new code, this is just a particularly flexible case).

Reply:
Okay, I will make that change

Reply:
Wow, you're right, `map_location` can be so many things!  :)